generated/nidigital/nidigital/session.py

```diff
@@ -2774,6 +2774,38 @@ def delete_all_time_sets(self):
         errors.handle_error(self, error_code, ignore_warnings=False, is_error_handling=False)
         return
 
+    @ivi_synchronized
+    def _file_helper(self, action, files):
```

Comment:
Can you give this a more descriptive name? like `_call_method_with_iterable`
rename `action` -> `method`

Reply:
Done